modules/campaignion_email_to_target_type/campaignion_email_to_target_type.info

```diff
@@ -23,7 +23,8 @@ dependencies[] = media
 dependencies[] = node
 dependencies[] = node_reference
 dependencies[] = opengraph_meta_upload
-dependencies[] = pgbar
+dependencies[] = pgbar:pgbar
+dependencies[] = pgbar:pgbar_webform
 dependencies[] = postcode
```

Comment:
Weren’t there more small fixes? :thinking: 

On the other hand, they would probably all belong in the 7.x-2.x branch as they are not `foundist` specific (this isn’t either).

Reply:
the other small fix was removing a permission from `edit mimemail user settings` (which is not set in this profile in the first place, so no need to remove). no other small fixes I am aware of...

Reply:
yes, can cherry pick the commit into the 7.x-2.x branch after approval...